dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/ClassLoadingServiceImpl.java

```diff
@@ -126,6 +129,16 @@ public void bundleChanged(BundleEvent event) {
     private final ConcurrentServiceReferenceSet<ClassGenerator> generatorRefs = new ConcurrentServiceReferenceSet<ClassGenerator>(REFERENCE_GENERATORS);
     private final ClassGeneratorManager generatorManager = new ClassGeneratorManager(generatorRefs);
 
+    @Reference(cardinality = ReferenceCardinality.MULTIPLE,
+               policy = ReferencePolicy.DYNAMIC,
+               policyOption = ReferencePolicyOption.GREEDY)
+    protected volatile List<ApplicationExtensionLibrary> appExtLibs;
```

Comment:
Can this be private?

Reply:
yes it can, but I believe that it is a best practice for all DS fields/methods to be protected visibility

Reply:
Thanks — I'll try to remember that